blueocean-core-js/src/js/capability/Capability.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Created by cmeyers on 9/8/16.
+ */
+export const CAPABILITY = {
+    SIMPLE_PIPELINE: 'org.jenkinsci.plugins.workflow.job.WorkflowJob',
```

Comment:
I removed this is lieu of changes in commit 	73d091a. I am not sure if this is exactly what you had in mind or if it's workable. We should probably define some of the core capabilities around pipelines in a place where dashboard and personalization can get at them to avoid "magic string" bugs.